my_site/blog/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+class Post(models.Model):
+    title = models.CharField(max_length=200, unique=True)
+    author = models.ForeignKey(User, on_delete= models.CASCADE,related_name='blog_posts')
```

Comment:
Lets keep PEP-8 code format (Space after comma, no space before or after '= ' in kwargs)